### ISSUE TITLE
Accordion Header requires clicking on title to open, can we make it open by clicking anywhere on the ngbPanelTitle? 

### ISSUE THREAD

**Issue by daBishMan** [ng-bootstrap/ng-bootstrap#1566]
Please remember, the issues forum is __NOT__ for support requests. It is for bugs and feature requests only.
Please read https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CONTRIBUTING.md and search
existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.

### Bug description:
Open the accordion panel by clicking anywhere on the header

### Link to minimally-working plunker that reproduces the issue:
http://plnkr.co/edit/RzBylRHu1J1B4HNJuzxI?p=preview

You can fork a plunker from one of our [demos](https://ng-bootstrap.github.io/#/components) and use it as a starting point.
Please note that we can _not_ act on bug reports without a minimal reproduction scenario in plunker. Here is why:
https://github.com/ng-bootstrap/ng-bootstrap#you-think-youve-found-a-bug

basically, I want to be able to open the panel without the need to click on the title, just by clicking anywhere on the ngbPanelTitle

### Version of Angular, ng-bootstrap, and Bootstrap:

Angular: 4.1.3

ng-bootstrap: alpah.26

Bootstrap: alpah.6

**Comment by pkozlowski-opensource** [comment]
This is duplicate of the existing issue #717

**Comment by daBishMan** [comment]
I do not think this is the same issue, I wan to be able to click anywhere and open the accordion, what I think 717 is some content opens it and some content does not

**Comment by pkozlowski-opensource** [comment]
@daBishMan it is exactly the same issue in the sense that both talk about which parts of the title are linked to the expand / collapse behaviour.
